### PR TITLE
[WIP] Refactor folder to be more pythonic

### PR DESCRIPTION
**Pull request by titilambert**
Here an example of what it could be.
You can already try this: 

```
pip install git+https://github.com/titilambert/ochopod.git@more_pythonic
```

I also remove some lines in Dockerfile...

With this change, you can also push ochopod on pypi...

What do you think about that ?

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -34,6 +34,7 @@
         'flask>=0.10.1',
         'kazoo>=2.2.1',
         'jinja2>=2.7.3',
+        'requests',
         'pykka>=1.2.0'
     ],
     package_data={
```
Comment (titilambert) [inline setup.py:40]: The "package_data" seems useless, can be deleted ?

**Comment by titilambert** [comment]
Documentation update is missing but I think it's mostly done

File: setup.py (hunk 1 above)
Comment (opaugam) [inline setup.py:40]: it's used to carry around the logging configuration.

**Comment by opaugam** [comment]
good stuff / let me pause a bit & merge this along with other changes after i run more high-level integration tests here (we use ochopod in production).

**Comment by opaugam** [comment]
btw Thibault, if you ever are interested to come have fun working for Autodesk... :-)

**Comment by titilambert** [comment]
Oki ! I'm continuing this PR with doc update

**Comment by opaugam** [comment]
hold on - i'm going to include this work in another push today :) I'll update the docs as well..

**Comment by titilambert** [comment]
Oups ... Ok ;)

**Comment by opaugam** [comment]
actually the README still has that old zookeeper-over-k8s example .. i'm going to turn it into a Mesos/DCOS example.

**Comment by titilambert** [comment]
OK, just tell me how I can help you ;)

**Comment by opaugam** [comment]
I'd be curious to see you using the ochothon PaaS and give me your impression. Do you have a running Mesos cluster on EC2 ?

**Comment by titilambert** [comment]
Not yet I'm just trying ochopod without mesos stack. Also, I know for sure I will not use Amazon EC2. So I think use ochopod with mesos (or kubernetes) in private infrastructure.
An other question, do you have any IRC chan ?

**Comment by titilambert** [comment]
Just keep me in touch, I have some other PRs to submit you ;)

**Comment by opaugam** [comment]
ok merged all of this + doc updates + a few other fixes

**Pull request closed without merging**